akka-sample-cqrs-java/src/main/resources/application.conf

```diff
@@ -36,11 +36,15 @@ akka {
 cassandra-journal {
   events-by-tag {
     bucket-size = "Day"
+    flush-interval = 50ms
+    pubsub-notification = on
+    refresh-interval = 2s
```

Comment:
Perhaps add a comment also noting that these are tweaked for the sake of latency?

Reply:
added